scripts/traits_logging.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+from traits.api import *
```

Comment:
`import *` is very bad.

http://docs.python-guide.org/en/latest/writing/structure/#packages
   
>It is possible to simulate the more standard behavior by using a special syntax of the import statement: from modu import *. This is generally considered bad practice. Using import * makes code harder to read and 
makes dependencies less compartmentalized.